Assets/Scripts/Models/Time/WorldTime.cs

```diff
@@ -294,13 +315,13 @@ public int Year
 
     #region IFormattable implementation
 
-    string IFormattable.ToString(string format, IFormatProvider provider = null)
+    string IFormattable.ToString(string format, IFormatProvider provider)
```

Comment:
As per the C# Language Specifications in 10.6.1:
"If optional parameters occur in an implementing partial method declaration (§10.2.7) , an explicit interface member implementation (§13.4.1) or in a single-parameter indexer declaration (§10.9) the compiler should give a warning, since these members can never be invoked in a way that permits arguments to be omitted."

Basically you aren't meant to add parameter options (like ref, optionals, params...) to interfaces if they don't have them, because it results in cases where you can't actually omit the parameter.  However, as per the IFormattable.ToString [method](https://msdn.microsoft.com/en-us/library/system.iformattable.tostring(v=vs.110).aspx) there is another function you can override or whatever that just has the format (that is `string IFormattable.ToString(string format)`) and normally it just calls the provider one with `CultureInfo.CurrentCulture` or null).  I can add this if you want?  Hopefully this however clears up why you shouldn't have default parameters on interface implementations :D.